test/models/RelationshipRecordTest.scala

```diff
@@ -26,11 +26,10 @@ import java.time.{LocalDate, LocalDateTime}
 class RelationshipRecordTest extends BaseTest with GuiceOneAppPerSuite {
 
   lazy val currentYear: Int = LocalDate.now().getYear
-  lazy val nextYear: Int = currentYear + 1
-  lazy val pastYear: Int = currentYear - 1
-  lazy val futureDateTime: String = "" + nextYear + "0101"
-  lazy val pastDateTime: String = "" + pastYear + "0101"
-  val dateFormat = DateTimeFormatter.ofPattern("yyyyMMdd")
+  lazy val futureDateTime: String = "" + currentTaxYear.finishYear + "0504"
```

Comment:
Do you need the empty string at the start? Could this just be `currentTaxYear.finishYear + "0504"` or `s"${currentTaxYear.finishYear}0504"`

Reply:
refactor: implemented string interpolation as suggested (for line 30 also)
Ty